conformance_tests/core/test_image/include/helpers_test_image.hpp

```diff
@@ -0,0 +1,32 @@
+/*
+ *
+ * Copyright (C) 2025 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ */
+
+#ifndef level_zero_tests_HELPERS_TEST_IMAGE_HPP
```

Comment:
You could make guards more verbose i.e.
`CONFORMANCE_TESTS_CORE_TEST_IMAGE_INCLUDE_HELPERS_TEST_IMAGE_HPP`